app/views/admins/applications/show.html.erb

```diff
@@ -0,0 +1,30 @@
+<% if @application.all_pets_approved %>
+  <h1>Pet Application For <%= @application.name %> Is Approved </h1>
+<% elsif @application.all_pets_approved == nil %>
+  <h1>Pending Pet Application For <%= @application.name %> </h1>
+  <p>Pets applied for:</p>
+  <ul>
+    <% @application.list_of_pets.each do |pet| %>
+    <div id="pet-<%= pet.id %>">
+      <li><%= pet.name %></li>
+        <ul>
+          <% if pet.application_pet_approved(params[:id]) %>
```

Comment:
There's some complex display logic here and you're not going to get rid of all of it (you will definitely have if statements here) but one option for consolidating some of this logic would be to display the status of the PetApplication dynamically. You could check if a PetApplication has a status, and if it does, just print the value with a label like so:
`<p>Status: <%= pet.application_pet.status %> </p>`